source/install.rst

```diff
@@ -76,20 +76,33 @@ If you don't have the data, then proceed to step 8 and 9
   
 .. code-block:: bash
 
-  ./bin/console fos:user:create --super-admin  
+  ./bin/console nines:create:user --super-admin  
   
-10. Install bower, npm, and nodejs if you haven't already. Then use bower to 
-   download and install the javascript and css dependencies.
+10. If you haven't installed npm and yarn globally, you will have to install them. You could do this by running the below commands in the terminal.
   
 .. code-block:: bash
 
-  bower install
+  sudo apt install npm
+  sudo npm install --global yarn
 
-11. Configure the web server. The application's `web/` directory must
+11. If you have installed npm and yarn globally, then set up yarn for this project by running the below command inside project directory.
+  
+.. code-block:: bash
+
+  yarn install
+
+12. Please follow the instructions in the config.rst file to set up the configuration settings for this project.
+
+13. Configure the web server. The application's `web/` directory must
     be accessible to the world. Symfony provides `example
     configurations`_ for most server setups.
 
+14. Start the Symfony server by using the below command and navigate to the link displayed.
   
+.. code-block:: bash
+
+  symfony server:start
```

Comment:
It's a good idea to include directions for stopping the server once it is up and running.